src/main/java/com/sleekbyte/tailor/utils/Configuration.java

```diff
@@ -57,16 +59,56 @@ public boolean shouldListFiles() {
         return CLIArgumentParser.shouldListFiles();
     }
 
+    /**
+     * Determine if the output should be colorized.
+     *
+     * @return whether the output should be colorized
+     */
     public boolean shouldColorOutput() {
-        return CLIArgumentParser.shouldColorOutput();
+        boolean shouldColorOutput = CLIArgumentParser.shouldColorOutput();
+        if (shouldColorOutput
+            && yamlConfiguration.isPresent()
+            && !yamlConfiguration.get().getColor().isEmpty()) {
+            String option = yamlConfiguration.get().getColor();
+            validateColorOption(option);
+            if (option.equals(Messages.DISABLE)) {
+                shouldColorOutput = false;
+            }
+        }
+        return shouldColorOutput;
     }
 
+    /**
+     * Determine if the output color should be inverted.
+     *
+     * @return whether the output color should be inverted
+     */
     public boolean shouldInvertColorOutput() {
-        return CLIArgumentParser.shouldInvertColorOutput();
+        boolean shouldInvertColorOutput = CLIArgumentParser.shouldInvertColorOutput();
+        if (!shouldInvertColorOutput
```

Comment:
this will not work if invert was set to true in the YAML file and false on the command line (the CLI should override the YAML file)